MekHQ/src/mekhq/campaign/parts/equipment/LargeCraftAmmoBin.java

```diff
@@ -169,18 +169,6 @@ protected Money getPricePerTon() {
         return Money.of(getType().getRawCost());
     }
 
-    @Override
-    public Money getStickerPrice() {
-        if (getShotsPerTon() <= 0) {
-            return Money.zero();
-        }
-
-        return getPricePerTon()
-                .multipliedBy(size)
-                .multipliedBy(getCurrentShots())
-                .dividedBy(getShotsPerTon());
-    }
-
```

Comment:
The real fix here is to just delete the `.multipliedBy(size)` line, which looks like a copy-paste error from `getValueNeeded`.

Compare this to AmmoBin.java:
https://github.com/MegaMek/mekhq/blob/785d0a3158866f7fdb8106c517d6899efc13bcc2/MekHQ/src/mekhq/campaign/parts/equipment/AmmoBin.java#L186-L195

Reply:
That would make the two methods the exact same, hence the deletion of the override.

Reply:
I missed it derived from AmmoBin in this case! Been playing with GitHub's code navigation features. Took me to EquipmentPart.

Reply:
@sixlettervariables No worries, my first version of the fix was exactly that, removing the line.  I cleaned up the override prior to opening the PR when I realized they would be the same.  Thanks for the review and merge, will try to package up some other changes I have lying around locally.